stack.yaml

```diff
@@ -108,6 +108,7 @@ extra-deps:
 - happy-1.19.5                    # https://github.com/commercialhaskell/stack/issues/3151
 - entropy-0.3.7                   # https://github.com/commercialhaskell/stack/issues/3151
 - cborg-0.1.1.0
+- systemd-1.1.2
```

Comment:
https://travis-ci.org/input-output-hk/cardano-sl/jobs/262601176#L2380

the `pkgs/generate.sh` has to be re-ran to add the `systemd` package under nix